contrib/debian/copyright

```diff
@@ -1,161 +0,0 @@
-Format: http://svn.debian.org/wsvn/dep/web/deps/dep5.mdwn?rev=174
```

Comment:
Also, we should leave all existing license notifications in place (for anything we have not deleted), and just add what we have included (see #3).